third_party/terraform/website/docs/r/storage_bucket.html.markdown

```diff
@@ -39,6 +39,43 @@ resource "google_storage_bucket" "image-store" {
 }
 ```
 
+```hcl
+resource "google_storage_bucket" "image-store" {
+  name     = "image-store.com"
+  location = "US"
+  force_destroy = true
+
+  bucket_policy_only = true
+
+  website {
+    main_page_suffix = "index.html"
+    not_found_page   = "404.html"
+  }
+  cors {
+    origin = ["http://image-store.com"]
+    method = ["GET", "HEAD", "PUT", "POST", "DELETE"]
+    response_header = ["*"]
+    max_age_seconds = 3600
+  }
+ }
+```
+
+```hcl
+resource "google_storage_bucket" "image-store" {
+  name     = "image-store-bucket"
```

Comment:
You might want to rename this bucket and resource to something more in line with the use case you are trying to show an example of. Maybe "auto-expiring-bucket" or something like that.

Reply:
Done